computer_vision/random_erasing_augmentation.py

```diff
@@ -0,0 +1,99 @@
+import glob
+import os
+import random
+from string import ascii_lowercase, digits
+import cv2
+
+
+# Params
+IMAGE_DIR = "1"
+OUTPUT_DIR = "out"
+MAX_RECTANGLE_WIDTH_PERCENT = 20  # (between 0 and 100)
+MAX_RECTANGLE_HEIGHT_PERCENT = 20  # (between 0 and 100)
+RECTANGLE_COLOR = "WHITE"  # ("BLACK" or "WHITE")
+
+
+def main() -> None:
+    """
+    Get images list from input dir.
+    Update new images.
+    Save images in output dir.
+    """
+    img_paths = get_dataset(IMAGE_DIR)
+    print("Processing...")
+    new_images, paths = update_images(
+        img_paths,
+        MAX_RECTANGLE_WIDTH_PERCENT,
+        MAX_RECTANGLE_HEIGHT_PERCENT,
+        RECTANGLE_COLOR,
+    )
+
+    for index, image in enumerate(new_images):
+        # Get random string code: '7b7ad245cdff75241935e4dd860f3bad'
+        letter_code = random_chars(32)
+        file_name = paths[index].split(os.sep)[-1].rsplit(".", 1)[0]
+        file_root = f"{OUTPUT_DIR}/{file_name}_NEW_{letter_code}"
+        cv2.imwrite(f"{file_root}.jpg", image, [cv2.IMWRITE_JPEG_QUALITY, 100])
+        print(f"Success {index+1}/{len(new_images)} with {file_name}")
+
+
+def get_dataset(img_dir: str) -> list:
+    """
```

Comment:
add doctest in `get_dataset`cfunction:-

    """

    Given the path to a folder that contains images, return a list of images path.
    >>> get_dataset('1')
    ['1/image1.jpg', '1/image2.jpg', '1/image3.jpg']

    """

Reply:
```
"""
>>> from pathlib import Path
>>> here = Path().resolve().parent
>>> get_dataset(here / "1")